packages/ui/src/views/repo/repo-sidebar/index.tsx

```diff
@@ -16,56 +14,58 @@ interface RepoSidebarProps {
   useTranslationStore: () => TranslationStore
 }
 
-export const RepoSidebar: FC<RepoSidebarProps> = ({
-  hasHeader,
-  hasSubHeader,
+export const RepoSidebar = ({
   navigateToNewFile,
   navigateToFile,
   filesList,
   children,
   selectBranchOrTag,
   useRepoBranchesStore,
   useTranslationStore
-}) => {
+}: RepoSidebarProps) => {
   const { branchList } = useRepoBranchesStore()
 
   return (
-    <SandboxLayout.LeftSubPanel className="w-[248px]" hasHeader={hasHeader} hasSubHeader={hasSubHeader}>
-      <SandboxLayout.Content className="flex h-full overflow-hidden p-0">
-        <div className="flex w-full flex-col gap-3 pt-5">
-          <div className="grid w-full auto-cols-auto grid-flow-col grid-cols-[1fr] items-center gap-2 px-5">
-            {branchList && (
-              <BranchSelector
-                onSelectBranch={selectBranchOrTag}
-                useRepoBranchesStore={useRepoBranchesStore}
+    // 100vh = screen height - (55px Breadcrumbs Height + 45px SubHeader Height = 100px)
+    // Total height of both the divs should be 100vh
+    <div className="sticky top-20 h-[calc(100vh-100px)]">
```

Comment:
as discussed, let's try removing fixed height from here. having the sidebar scroll the page is not a bad idea. it looks pretty good to me without a fixed height

Reply:
As discussed, we may remove this after confirming with the XD team.